src/epubParsers.js

```diff
@@ -1,113 +1,110 @@
 import AWS from 'aws-sdk'
 import unzip from 'unzip-stream'
-import stream from 'stream'
-import {resultHandler} from './responseHandlers'
+import ResHandler from './responseHandlers'
 
 AWS.config.update({
-    region: 'us-east-1',
-    logger: process.stdout
+  region: 'us-east-1',
+  logger: process.stdout
```

Comment:
we'll need to add a logger like winston for sending standard log messages to CloudWatch

Reply:
I've added Winston as the logger for the function but not on this line. I think that this output should be reflected in CloudWatch, but I need to look at how best to integrate winston with the AWS SDK. There is at least one library that does so, but I haven't reviewed it yet